code/src/java/pcgen/io/PCGIOHandler.java

```diff
@@ -257,7 +257,7 @@ public void read(PlayerCharacter pcToBeRead, InputStream in,
 		}
 	}
 
-	private boolean isPCGCersion2(List<String> lines)
+	private static boolean isPCGCersion2(List<String> lines)
```

Comment:
I'd leave them as instance methods unless there is a notable benefit to having them static.